integration_tests/mockData/prisonerByIdData.ts

```diff
@@ -8,11 +8,15 @@ const prisoners = {
       status: 200,
       headers: { 'Content-Type': 'application/json;charset=UTF-8' },
       jsonBody: {
+        prisonId: 'MDI',
         prisonerNumber: 'A00001A',
         firstName: 'Paris',
         lastName: 'Jones',
-        releaseDate: new Date().toISOString(),
+        releaseDate: '2030-12-31',
         nonDtoReleaseDateType: 'HDC',
+        receptionDate: '2001-17-01',
```

Comment:
Our wiremock stubs in our integration tests were missing a few fields from `prisoner-search-api`